NEWS.md

```diff
@@ -1,3 +1,15 @@
+# ss3sim 1.19.0
+
+* Bump version to match current version of SS3,
+  where version numbers of ss3sim will now be major version of ss3sim,
+  followed by minor version number of SS3, and terminated with minor version
```

Comment:
Maybe clarify that "minor version" means the last third set of digits in the ss3 versioning?

Also, what happens if the 30 part of SS3 gets incremented? I don't see this coming in the near future but it could happen.... Maybe we should use 3019 as the ss3 number instead of 19? Not sure...

Reply:
To try and keep things less complicated I just left it as is but I did increase the notes wrt what the minor version of SS3 means. I think we can just increment our major versions when SS3 becomes SS3.31.x

Changed in 7915f4384fe923ae9aad836fa934c45c585de4d6